libs/agno/agno/eval/accuracy.py

```diff
@@ -453,5 +457,16 @@ def run(
         if self.print_summary or self.print_results:
             self.result.print_summary(console)
 
+        # Track results
+        if self.track_results:
+            track_eval_run(
+                run_id=self.eval_id,  # type: ignore
+                run_data=asdict(self.result),
+                eval_type=EvalType.ACCURACY,
+                agent_id=self.agent.agent_id if self.agent is not None else None,
+                agent_name=self.agent.name if self.agent is not None else None,
+                user_id=self.agent.user_id if self.agent is not None else None,
```

Comment:
This is the user that is using the app, not the user of the API (i.e. the developer). It doesn't make sense to include here.